### ISSUE TITLE
Video display beyond app frame

### ISSUE THREAD

**Issue by kushwahbhavesh** [waydroid/waydroid#325]
**Bug**
When playing from any app the video goes beyond app frame or runs in app frame.

**Screenshots**
![Screenshot from 2022-03-12 18-19-02](https://user-images.githubusercontent.com/101459744/158018634-ffdc34e0-a2eb-4955-9897-d481dc393131.png)
![Screenshot from 2022-03-12 18-10-14](https://user-images.githubusercontent.com/101459744/158018398-80b62a72-d14d-46d7-bb19-6a0326a0a4fa.png)
![Screenshot from 2022-03-12 18-10-36](https://user-images.githubusercontent.com/101459744/158018415-039d748a-fb1d-4a9a-85e5-d43746d328ba.png)

**General information**
 - Waydroid tools Version: 1.2.0
 - Waydroid Images Version: VANILLA 20211021

**Desktop**
 - OS: Fedora Linux 35 (Workstation Edition) x86_64
 - GPU: Intel 4th Gen Core Processor
 - Kernel version: 5.16.12-200.fc35.x86_64
 - Desktop: GNOME 41.4

**Comment by kjay23** [comment]
Same here, I couldn't get any proper video playback with any resolutions and audio has small latency. Because Waydroid is a forked version of Anbox, it has still the same weird issues (but few of them are mostly fixed). It still lacks hardware decoding rather than software so as you expect from this [article](https://github.com/anbox/anbox/issues/956#issuecomment-743721771) only Firefox browser works just fine but with small hiccups and yeah, Waydroid is still good at least. ;)

**Comment by kushwahbhavesh** [comment]
Yeah it's good, i have no problem with any audio, only the problem is with video, other thing just works great.

**Comment by kushwahbhavesh** [comment]
> Same here, I couldn't get any proper video playback with any resolutions and audio has small latency. Because Waydroid is a forked version of Anbox, it has still the same weird issues (but few of them are mostly fixed). It still lacks hardware decoding rather than software so as you expect from this [article](https://github.com/anbox/anbox/issues/956#issuecomment-743721771) only Firefox browser works just fine but with small hiccups and yeah, Waydroid is still good at least. ;)

Yeah it's good, i have no problem with any audio, only the problem is with video, other thing just works great.
